packages/cozy-client/src/CozyClient.js

```diff
@@ -288,7 +301,7 @@ export default class CozyClient {
     this.ensureStore()
     const existingQuery = getQueryFromState(this.store.getState(), queryId)
     // Don't trigger the INIT_QUERY for fetchMore() calls
-    if (existingQuery.fetchStatus !== 'loaded' || !queryDefinition.skip) {
+    if (existingQuery.fetchStatus !== 'loaded') {
```

Comment:
I don't think this modification was intended to land in this PR. This is related to https://github.com/cozy/cozy-client/pull/139, that is finally not required (see https://github.com/cozy/cozy-client/pull/139#issuecomment-418795795).